ci-operator/step-registry/ipi/deprovision/vsphere/dns/ipi-deprovision-vsphere-dns-ref.yaml

```diff
@@ -0,0 +1,10 @@
+ref:
+  as: ipi-deprovision-vsphere-dns
+  from: upi-installer
+  commands: ipi-deprovision-vsphere-dns-commands.sh
+  resources:
+    requests:
+      cpu: 1000m
+      memory: 100Mi
+  documentation: >-
+    Releases DNS records via `${SHARED_DIR}/dns-delete.json`.  The IP addresses were allocated with the ipi-conf-vsphere-dns step.
```

Comment:
mention the `${SHARED_DIR}/hosted-zone.tx`

Reply:
> mention the `${SHARED_DIR}/hosted-zone.tx`

Done with 71b216028 -> a466b8351.